apps/nextjs/src/app/(dashboard)/dashboard/_components/sidebar/nav-main.tsx

```
@@ -98,10 +52,83 @@ interface NavMainItem {
+  );
+}
+
+function EmptyNavItems() {
+  return (
+    <SidebarGroup>
+      <SidebarGroupLabel aria-label="Navigation Menu">
+        Platform
+      </SidebarGroupLabel>
+      <div
+        className="px-2 py-4 text-sm text-muted-foreground"
+        role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)